.storybook/main.js

```diff
@@ -1,21 +1,26 @@
-const path = require('path');
```

Comment:
for some reason some files still missing `;` and trailing commas

Reply:
will check!

Reply:
Should be good now...I hope :D